src/index.css

```diff
@@ -12,19 +12,18 @@
   margin-bottom:.81696rem;
   margin-top:0
  }
- .dashboard-link,h1,small {
-  line-height:1.07143rem;
-  padding-top:.25447rem;
-  margin-bottom:.81696rem;
-  margin-top:0
- }
  * {
   box-sizing:border-box
  }
- body,html {
-  margin:0;
-  padding:0;
-  height:100%
+ html, body {
+  overflow-x: hidden;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: 0;
+  padding: 0;
```

Comment:
There is zero-pixel overflow, which triggered the horizontal scrollbar, which is now gone.